process-engine/process_engine_runtime
# ✅ Cleanup databases after each test run

## Changes

1. Clear out all databases after each test
    - This replaces the test's individual cleanup steps
2. Fix an issue with the KPI API test that caused the SQLite tests to crash
3. Improve some test case wordings
4. Fix reference error in `ConsumerApi.GetUserTasksForCorrelation` test

## Issues

Closes #394 

PR: #403

## How to test the changes

- Run the tests
- See if anything remains in the test databases after the tests are done
